modules/apps/site/site-cms-site-initializer/src/main/java/com/liferay/site/cms/site/initializer/internal/display/context/ContentSectionDisplayContext.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.site.cms.site.initializer.internal.display.context;
+
+import com.liferay.frontend.data.set.model.FDSActionDropdownItem;
+import com.liferay.frontend.taglib.clay.servlet.taglib.util.DropdownItem;
+import com.liferay.petra.string.StringBundler;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Sam Ziemer
+ */
+public class ContentSectionDisplayContext {
+
+	public String getAPIURL() {
+		StringBundler sb = new StringBundler();
+
+		sb.append("/o/search/v1.0/search?emptySearch=true");
+		sb.append("&nestedFields=embedded&entryClassNames=");
+
+		for (int i = 0; i < _CONTENT_CLASS_NAMES.length; i++) {
+			sb.append(_CONTENT_CLASS_NAMES[i]);
+
+			if (i != (_CONTENT_CLASS_NAMES.length - 1)) {
+				sb.append(",");
+			}
+		}
+
+		return sb.toString();
+	}
+
+	public List<DropdownItem> getBulkActionDropdownItems() {
+		return new ArrayList<>();
+	}
+
+	public List<FDSActionDropdownItem> getFDSActionDropdownItems() {
+		return new ArrayList<>();
+	}
+
+	private static final String[] _CONTENT_CLASS_NAMES = {
```

Comment:
@SamZiemer, I'm a bit unsure about having the classNames here (even though we talked about it). There might be cases were we want to take into account other entry types and we would have to add them by hand here

Reply:
Thanks @jkappler. We discussed in slack, but I'll add it here for transparency. I had originally thought I should use the class, like BlogsEntry.class.getName() for each class we want, however that would require a dependency for each object type we want to add and I was not sure if that was the correct route. @robertoDiaz, what do you think, is there a better way or cleaner approach to either the current one in this PR, or using the classes?

Reply:
I've agreed with Jurgen adding a system setting similar to `com.liferay.portal.security.antisamy.configuration.AntiSamyConfiguration#whitelist` to set this values